shop/shop/src/main/java/ro/msg/learning/shop/Mapper.java

```diff
@@ -0,0 +1,43 @@
+package ro.msg.learning.shop;
+
+import org.springframework.stereotype.Component;
+import ro.msg.learning.shop.dto.ProductCategoryDto;
+import ro.msg.learning.shop.dto.ProductDto;
+import ro.msg.learning.shop.dto.SupplierDto;
+import ro.msg.learning.shop.model.Product;
+import ro.msg.learning.shop.model.ProductCategory;
+import ro.msg.learning.shop.model.Supplier;
+import java.math.BigDecimal;
+
+@Component
+public
+class Mapper {
```

Comment:
The name should be more specific, I see you have product mappings in here so you should call it `ProductMapper`. Also it should not stay at the root level, you should move it to a folder called `mapper` or `mappers`. Also you have a new line after `public`, it should be right before class